guides/common/attributes-orcharhino.adoc

```diff
@@ -47,7 +47,6 @@
 :smartproxy-example-com: orcharhino-proxy.example.com
 :smartproxy_port: 9090
 :fdi-package-name: orcharhino-fdi
-:project-debug: orcharhino-debug
```

Comment:
I have no idea what this package *exactly* does, but we stopped shipping katello-debug on EL9 because it was broken there, and I'd expect this to be broken as a result too.

Reply:
Works for me!